build.gradle

```diff
@@ -94,6 +94,7 @@ spotless {
 		importOrder 'java', 'javax', 'org', 'com', 'dev.jbang', ''
 		removeUnusedImports()
 		eclipse().configFile "misc/eclipse_formatting_nowrap.xml"
```

Comment:
Aaah, I had put my exclude in the wrong section. Duh.

Reply:
Yeah. I was getting upset why it wasn't working ...until i noticed there was multiple sections :)